qa/tasks/mgr/dashboard/test_health.py

```diff
@@ -99,7 +99,7 @@ def test_full_health(self):
                         'percent_used': float,
                         'kb_used': int,
                         'quota_bytes': int,
-                        'raw_bytes_used': int,
+                        'stored_raw': int,
```

Comment:
@ifed01 this represents the schema of the df output, so i think we need to add the newly added fields, like `stored`, `compress_bytes_used` and `compress_under_bytes`.